src/moonbeam/moonbase.js

```diff
@@ -0,0 +1,217 @@
+import _ from 'lodash';
+import Keyring from '@polkadot/keyring';
+import BN from 'bn.js';
+// import moment from 'moment';
+import { hexToU8a, u8aToHex } from '@polkadot/util';
+
+import Account from '../common/account';
+import { TuringDev, MoonbaseDev } from '../config';
+import TuringHelper from '../common/turingHelper';
+import MoonbaseHelper from '../common/moonbaseHelper';
+import {
+    sendExtrinsic, getDecimalBN,
+    // listenEvents, calculateTimeout,
+} from '../common/utils';
+
+// TODO: read this instruction value from Turing Staging
+// One XCM operation is 1_000_000_000 weight - almost certainly a conservative estimate.
+// It is defined as a UnitWeightCost variable in runtime.
+const TURING_INSTRUCTION_WEIGHT = 1000000000;
+const MIN_BALANCE_IN_PROXY = 10; // The proxy accounts are to be topped up if its balance fails below this number
+// const TASK_FREQUENCY = 3600;
+
+const WEIGHT_PER_SECOND = 1000000000000;
+
+const keyring = new Keyring({ type: 'sr25519' });
+
+const sendXcmFromMoonbase = async ({
+    turingHelper, parachainHelper, turingAddress,
+    paraTokenIdOnTuring, keyPair,
+}) => {
+    console.log('\na). Create a payload to store in Turing’s task ...');
+    const taskExtrinsic = turingHelper.api.tx.system.remarkWithEvent('Hello!!!');
```

Comment:
I just found that `system.remarkWithEvent` does not emit an event even executed on our chain. It’s not a good call to try. Please just use our `automationTime.scheduleDynamicDispatchTask` directly. At least you could see a clear event in Network.

Reply:
Fixed